client/core/types.go

```diff
@@ -219,7 +219,15 @@ type OrderBook struct {
 	Epoch []*MiniOrder `json:"epoch"`
 }
 
+// MarketOrderBook is used as the BookUpdate's Payload with the FreshBookAction.
+// The subscriber will likely need to translate into a JSON tagged type.
+type MarketOrderBook struct {
```

Comment:
Could also have updated the `OrderBook` type with the base/quote fields, but that was looking to be a hairy and error prone change.